bower.json

```diff
@@ -28,6 +29,7 @@
     "raven-js": "^3.17.0"
   },
   "resolutions": {
-    "angular": ">= 1.3.13"
+    "angular": ">= 1.3.13",
+    "bip70-js": "^1.0.6"
```

Comment:
did bower do this at one stage? not sure what this section does, but the versions conflict